packages/next/src/build/index.ts

```diff
@@ -1108,7 +1108,8 @@ export default async function build(
                 config,
                 distDir,
                 pageKeys,
-                pageInfos: [],
+                // TODO: Is this wrong?
```

Comment:
do we still need this comment?
```suggestion
```

Reply:
Yeah, we still need to check if this is wrong, I'm inclined to say that it is wrong currently and we have to fix it.